pkgs/development/python-modules/pyrp3/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, myhdl
+, rpyc
+, cached-property
+, numpy
+}:
+
+buildPythonPackage {
+  pname = "pyrp3";
+  version = "unstable-2022-05-09";
+  # There's no pypi source
+  src = fetchFromGitHub {
+    owner = "linien-org";
+    repo = "pyrp3";
+    rev = "f13da68d825ede3091a082edf99339c5ed736bd2";
+    hash = "sha256-hmPG7jBPUSVnsLZOS0pf2c5gibWx0al0rAso1OTAZ1E=";
+  };
+
+  postPatch = ''
+    sed -i 's#self\.prefix#"${placeholder "out"}"#' setup.py
+  '';
+
+  postBuild = ''
+    python setup.py lib_build
+  '';
+  preInstall = ''
+    python setup.py lib_install
+  '';
+  # No tests
+  doCheck = false;
```

Comment:
`pythonImportsCheck` though?

Reply:
> `pythonImportsCheck` though?

Thanks for the reminder, it pointed out that there were issues in the result.